roles/openshift_node/tasks/config/override.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+LimitNOFILE=65535
```

Comment:
This should be in `roles/openshift_node/tasks/files/`.  If you put it there, you will not need  to specify the directory in the `copy` task in the role.